modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ItemActionForm.tsx

```diff
@@ -332,6 +353,36 @@ const ItemActionForm = ({
 								/>
 							</ClayForm.Group>
 
+							<ClayForm.Group>
+								<label htmlFor={permissionKeyFormElementId}>
+									{Liferay.Language.get(
+										'headless-action-key'
```

Comment:
Risk of inconsistency in naming :warning:

Reply:
In DSM persistence and in FDS this is referred to as `permissionKey`. I figured the least painful inconsistency here is to have `permissionKey` everywhere in code, and `headless action key` only in end label. Later on, we it might make sense to refactor code everywhere. Although, to me, `headless action key` in UI seems kinda misleading and confusing, as is not strictly related to headless actions in UI.